yarn.lock

```diff
@@ -2473,6 +2473,11 @@ class-utils@^0.3.5:
     isobject "^3.0.0"
```

Comment:
Make sure to add `yarn.lock` to `.gitignore`.

Reply:
❗ This does not belong to `.gitignore`. It should be in the repo, to make sure everyone (including the computer running the build in production/staging) is running the same version of every dependency

Reply:
Alright, thanks for the clarification!